src/components/Select/Select.tsx

```diff
@@ -27,6 +27,7 @@ export type SelectProps = Omit<
   error?: boolean | string;
   allowAutoFill?: boolean;
   shiftIconLeftwards?: boolean;
+  optionsOpening?: 'down' | 'up';
```

Comment:
Just thinking on making this a bit more self-explanatory, maybe `dropDirection`?
I think the most common nomenclature for this is 'dropdown'/'dropup'.

Reply:
`openingDirection` also makes sense